kitsune/sumo/static/sumo/scss/components/_banner.scss

```diff
@@ -0,0 +1,110 @@
+@use '../config' as c;
+@use '../protocol' as p;
+
+// Banners
+//
```

Comment:
how about naming this `_cta_banner.scss` in case we have another in the future?

Reply:
I wrote the CSS to be as flexible as possible - it'll function without a CTA button, without a close button, and/or without an image - so it's a generic banner, even though we're using it as a CTA banner in this instance.